app/src/main/java/com/google/android/samples/socialite/ui/chat/ChatScreen.kt

```diff
@@ -119,6 +128,8 @@ fun ChatScreen(
     onVideoClick: (uri: String) -> Unit,
     prefilledText: String? = null,
     viewModel: ChatViewModel = hiltViewModel(),
+    sharedTransitionScope: SharedTransitionScope,
+    animatedContentScope: AnimatedContentScope
```

Comment:
You may want to consider using Composition locals here if the hierarchy is very deep and you find yourself passing these scopes around quite a bit. https://developer.android.com/develop/ui/compose/animation/shared-elements#understand-scopes